Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -37,10 +47,14 @@ def csv_file_to_indicator_list(file_path, col_num, starting_row, auto_detect, de
 
             line_index = line_index + 1
 
+            if limit and len(indicator_list) == int(str(limit)):
```

Comment:
do we need the double casting to `str` and then to `int`?

Reply:
returns a mypy error otherwise